CMakeLists.txt

```diff
@@ -171,8 +171,11 @@ install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
 )
 
 # Enable unit testing by including the `test` subdirectory.
-enable_testing()
-add_subdirectory(test EXCLUDE_FROM_ALL)
+option(BUILD_TESTING "Enable building tests" ON)
+if(BUILD_TESTING)
+  enable_testing()
+  add_subdirectory(test EXCLUDE_FROM_ALL)
+endif()
```

Comment:
I saw that there was a `BUILD_TESTING` flag being used, but it didn't seem to affect the build.  I needed to build without tests (I didn't have python 3.7 installed), so I added this option.

Feel free to suggest a different name for the flag.

Reply:
[`BUILD_TESTING` is automatically set when we include `CTest`](https://cmake.org/cmake/help/v3.7/module/CTest.html). I think the issue is that `CTest` is being included in `test/examples/CMakeLists.txt` (line: 1). Let's replace line 174 with `include(CTest)` and remove line 1 of `test/examples/CMakeLists.txt`.

Reply:
Does this look like what you had in mind?